.vscode/settings.json

```diff
@@ -18,7 +18,7 @@
   "files.trimTrailingWhitespace": true,
   "solidity.packageDefaultDependenciesContractsDirectory": "contracts",
   "solidity.packageDefaultDependenciesDirectory": "lib",
-  "solidity.compileUsingRemoteVersion": "v0.8.17+commit.8df45f5f",
+  "solidity.compileUsingRemoteVersion": "v0.8.24+commit.e11b9ed9",
```

Comment:
Let's revert this change for now. Will redo the global extension settings in a separate PR.